boards/arm/stm32l562e_dk/stm32l562e_dk.dts

```diff
@@ -21,12 +21,37 @@
 		zephyr,flash = &flash0;
 	};
 
+	power-states {
+		stop0: state0 {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			substate-id = <0>;
+			min-residency-us = <100>;
+		};
+		stop1: state1 {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			substate-id = <1>;
+			min-residency-us = <500>;
+		};
+		stop2: state2 {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			substate-id = <2>;
+			min-residency-us = <900>;
+		};
+	};
```

Comment:
Please move this to `boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi`

Reply:
ok, moved to boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi